### ISSUE TITLE
Option to add text stroke

### ISSUE THREAD

**Issue by bbtimx** [chartjs/chartjs-plugin-datalabels#31]
It would be nice to add the option to show text stroke.

I now modify the code after line 402 to hard implement this:

```
    ctx.font = model.font.string;
    ctx.fillStyle = color;
    ctx.textAlign = align;
    ctx.textBaseline = 'middle';
    ctx.lineWidth = 4;
    ctx.lineJoin = 'round';
    ctx.miterLimit = 0;
    ctx.strokeStyle = '#ffffff';

    for (i = 0; i < ilen; ++i) {
        ctx.strokeText(
            lines[i],
            Math.round(x),
            Math.round(y),
            Math.round(rect.w));
        ctx.fillText(
            lines[i],
            Math.round(x),
            Math.round(y),
            Math.round(rect.w));

        y += lh;
    }
```

**Comment by simonbrunel** [comment]
Good idea, I need to think about the API

**Comment by simonbrunel** [comment]
I'm thinking to expose the following options:

- `textStrokeColor` (mapped to context.strokeStyle)
- `textStrokeWidth` (mapped to context.lineWidth)

And maybe deprecate `color` in favor of `textColor`

**Comment by simonbrunel** [comment]
@bbtimx I tried to implement text stroke, however I'm getting ugly results, especially with small font size (< 20px). I also tried to call `fillText` before `strokeText` but it's even worse, so I'm not sure to integrate this feature.

Can you share a screenshot and a jsfiddle of your use case with your implementation?

**Comment by bbtimx** [comment]
My config:

```
plugins: {
  datalabels: {
    anchor: 'end',
    color: '#991d85',
    font: {
      size: 14,
      style: 'bold'
    }
  }
}
```
Screenshot:
![image](https://user-images.githubusercontent.com/4718129/42966233-ca1f3d8a-8b9c-11e8-8ec4-cfcd8977cab6.png)

My implementation is still exactly like above.

**Comment by simonbrunel** [comment]
Finally implemented as part of b3c0faf33fdea2 (`textStrokeColor` & `textStrokeWidth`). @bbtimx If you still use this plugin, can you please checkout and build master and test this feature? I'm planning to release 0.5.0 in the next few days and would like to be sure it works as expected

**Comment by simonbrunel** [comment]
Released in [v0.5.0](/chartjs/chartjs-plugin-datalabels/releases/tag/v0.5.0)
